Fill.py

```diff
@@ -931,92 +907,102 @@ def failed(warning: str, force: typing.Union[bool, str]) -> None:
                 locations.remove("non_early_locations")
                 for target_player in worlds:
                     locations += non_early_locations[target_player]
-
-            block['locations'] = list(dict.fromkeys(locations))
-
-            if not block['count']:
-                block['count'] = (min(len(block['items']), len(block['locations'])) if
-                                  len(block['locations']) > 0 else len(block['items']))
-            if isinstance(block['count'], int):
-                block['count'] = {'min': block['count'], 'max': block['count']}
-            if 'min' not in block['count']:
-                block['count']['min'] = 0
-            if 'max' not in block['count']:
-                block['count']['max'] = (min(len(block['items']), len(block['locations'])) if
-                                         len(block['locations']) > 0 else len(block['items']))
-            if block['count']['max'] > len(block['items']):
-                count = block['count']
-                failed(f"Plando count {count} greater than items specified", block['force'])
-                block['count'] = len(block['items'])
-            if block['count']['max'] > len(block['locations']) > 0:
-                count = block['count']
-                failed(f"Plando count {count} greater than locations specified", block['force'])
-                block['count'] = len(block['locations'])
-            block['count']['target'] = multiworld.random.randint(block['count']['min'], block['count']['max'])
-
-            if block['count']['target'] > 0:
-                plando_blocks.append(block)
+            for target_player in worlds:
+                for group in multiworld.worlds[target_player].location_name_groups:
+                    if group in locations:
+                        locations.extend(multiworld.worlds[target_player].location_name_groups[group])
```

Comment:
This runs into problems if one world's location name matches another world's group name. I don't really know how you want to solve this, *maybe* requiring a prepended `group_` could work? But I do think it is a bug that should be fixed